File: TryCatch/app/src/main/java/com/example/exception/MainActivity.java
package com.example.exception;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        try {
            onAction();
        } catch (Exception sdfg) {
            System.out.println("onAction에서 6을 발견했나 봅니다.");
        } finally {
            System.out.println("try-catch문을 종료하고 finnaly문을 처리합니다.");
        }
    }

    public static void onAction() throws Exception {
        for (int i = 0; i < 10; i++) {
            try {
                if (i != 6) {
                    System.out.println(i+"는 6이 아닙니다");
                }else{
                    throw new Exception();
                }
            }catch(Exception sacaca){
                System.out.println(i+"는 6이 맞네요. CheckException이 onAction()서 발생");

            }
        }
    }
}

File: KotlinActivity/app/src/main/java/com/puresoftware/kotlinactivity/MainActivity.kt
package com.puresoftware.kotlinactivity

import android.os.Build
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import androidx.annotation.RequiresApi
import com.puresoftware.kotlinactivity.GgangSamKotlin.DataClassActivity
import java.lang.Exception

class MainActivity : AppCompatActivity() {
    @RequiresApi(Build.VERSION_CODES.N)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)



    }
}
